src/app/all-recipes/all-recipes.component.html

```diff
@@ -0,0 +1,58 @@
+<button routerLink="/create">Create new recipe</button>
+<div class="book-image">
+  <div class="book-sector">
+    <div class="create-recipe-section-left scrollbar">
+      <div class="recipe" *ngFor="let recipe of allRecipies">
```

Comment:
Instead of "recipe" class maybe you should consider "recipe-list-sector" or something like this.